homeassistant/components/huum/strings.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
I noticed there are differences between `strings.json` and `en.json` which I think need to be in sync.